cluster-autoscaler/cloudprovider/clusterapi/deployment.yaml

```diff
@@ -0,0 +1,158 @@
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: cluster-autoscaler
+  namespace: kube-system
```

Comment:
This should probably not be kube-system by default. It might be good to use a variable for this and expect users to run envsubst or similar against this prior to deploying.